packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -176,6 +176,10 @@ export enum ScreenShareFloorStatus {
   RELEASED = 'floor_released',
 }
 
+enum StreamEvents {
+  PublishedStateUpdate = 'published-state-update',
```

Comment:
`PUBLISHED_STATE_UPDATE = 'published-state-update'`  -  To keep it consistent with other enums

Reply:
hmm I think if we really want to be consistent with other events emitted by the SDK's Meeting object, then this should be added to EVENT_TRIGGERS and not as a new separate `StreamEvents` enum and it should be something like this:
```
MEETING_LOCAL_STREAM_PUBLISHED: 'meeting:localStreamPublished',
MEETING_LOCAL_STREAM_UNPUBLISHED: 'meeting:localStreamUnpublished',
```
as all existing meeting events seem to be done that way that we have a separate "started" and "stopped" event. Although personally I think I would prefer just a single one like this:
```
MEETING_STREAM_PUBLISH_STATE_CHANGED: 'meeting:streamPublishStateChanged',
```

Reply:
Agreed with Marcin's comment. I'll go ahead and update the event name to `MEETING_STREAM_PUBLISH_STATE_CHANGED: 'meeting:streamPublishStateChanged'` and will add it to EVENT_TRIGGERS.

Reply:
Done.